polaris-react/src/components/ResourceList/ResourceList.scss

```diff
@@ -222,33 +186,23 @@ $item-wrapper-loading-height: 64px;
   z-index: resource-list(bulk-actions-wrapper-stacking-order);
   width: 100%;
 
-  // se23: This is the initial value of flex, is there something else that mutates this in other components?
-  @media #{$p-breakpoints-sm-up} {
-    flex: 0 1 auto;
-    align-self: flex-start;
```

Comment:
The specificity of this `@media #{$p-breakpoints-sm-up}` matches the `#{$se23} &` below, but `#{$se23} &` wins because it comes last. Therefore, `align-self: flex-start` never gets applied and so can now be deleted 👍